### DISCUSSION TITLE
EasyMCP2221.exceptions.NotAckError: Device did not ACK.

### DISCUSSION THREAD

**Discussion by KelvinSan** [electronicayciencia/EasyMCP2221#17] [Q&A]
Question is this library compatible with the BME680 or BME688, everytime I run the example i get this error 

    raise NotAckError("Device did not ACK.")
EasyMCP2221.exceptions.NotAckError: Device did not ACK.

Thanks for your time in advance

**Comment by electronicayciencia** [comment]
I don't have that chip, so I don't know. But I don't see any reason why it wouldn't be supported.
Check [Basic weather station](https://easymcp2221.readthedocs.io/en/latest/smbus.html#example-1-basic-weather-station) for an example with a BME280.
